appshell/cefclient_gtk.cc

```diff
@@ -111,9 +111,11 @@ void TerminationSignalHandler(int signatl) {
   MainContext::Get()->GetRootWindowManager()->CloseAllWindows(true);
 }
 
-int RunMain(int argc, char* argv[]) {
+int RunMain(int argc, char** argv) {
```

Comment:
might not be really needed in this PR's context

Reply:
@abhijitapte I will revert this change.